modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -86,38 +97,32 @@ protected function runIt() {
       return $this->setResultError(sprintf('Unable to resolve file name "%s" for resource with identifier "%s" and version "%s".', $file_path, $this->resource->getIdentifier(), $this->resource->getVersion()));
     }
 
-    // Read the header line from the CSV file.
+    // Read the columns and EOL character sequence from the CSV file.
     try {
-      $header_line = $this->getFirstLineFromFile($file_path);
+      ['columns' => $columns, 'lines' => $column_lines] = $this->getColsFromFile($file_path);
```

Comment:
```suggestion
      [$columns, $column_lines] = $this->getColsFromFile($file_path);
```
Can we simplify this?